stdlib/std/contract.tact

```diff
@@ -19,3 +19,21 @@ native newAddress(chain: Int, hash: Int): Address;
 asm fun myAddress(): Address { MYADDR }
 
 asm fun myBalance(): Int { BALANCE FIRST }
+
+asm fun sendMessage(msg: Cell, mode: Int): Int { SENDMSG }
+
+asm fun gasConsumed(): Int { GASCONSUMED }
+
+asm(gas_used workchain) fun getComputeFee(workchain: Int, gas_used: Int): Int { GETGASFEE }
+
+asm(cells bits seconds workchain) fun getStorageFee(workchain: Int, seconds: Int, bits: Int, cells: Int): Int { GETSTORAGEFEE }
+
+asm(cells bits workchain) fun getForwardFee(workchain: Int, bits: Int, cells: Int): Int { GETFORWARDFEE }
+
+asm(gas_used workchain) fun getSimpleComputeFee(workchain: Int, gas_used: Int): Int { GETGASFEESIMPLE }
+
+asm(cells bits workchain) fun getSimpleForwardFee(workchain: Int, bits: Int, cells: Int): Int { GETFORWARDFEESIMPLE }
+
+asm(fwd_fee workchain) fun getOriginalFwdFee(workchain: Int, fwd_fee: Int): Int { GETORIGINALFWDFEE }
+
+asm fun myStorageDue(): Int { DUEPAYMENT }
```

Comment:
1. Let's add comments explaining what all these functions do.
2. Do we want to do arguments shuffling? This increases gas usage. And, for instance, HipoFinance don't do this shuffling, so porting their contracts to Tact would be slightly harder and error-prone.